.circleci/config.yml

```diff
@@ -310,6 +321,9 @@ workflows:
       - metrics_capture:
           requires:
             - cargo_fetch
+          filters:
+            branches:
+              only: master
```

Comment:
maybe enable it for this branch as well, temporarily, so we can see it in action before merging?

Reply:
Ask and ye shall receive:

https://circleci.com/gh/filecoin-project/rust-fil-proofs/24885

scroll down to `Run micro benchmarks on remote box` and then `Run ZigZag benchmarks on remote host with 1GiB sector size` sections